frontend/src/pages/MapView/index.jsx

```diff
@@ -1,80 +1,13 @@
 import React, { Component } from "react";
 import ReactMapGL, { Marker, Popup } from "react-map-gl";
 
+import api from "../../utils/api";
```

Comment:
Let's do an expansion of these calls using `{ }` syntax, like ` { getSearchResults }`.